### ISSUE TITLE
[UWP] Could not load assembly when running Migration

### ISSUE THREAD

**Issue by allanrsmith** [dotnet/efcore#6556]
### Steps to reproduce

Create a UWP Application project
Create a Universal Windows library project
- Add EntityFramework to the library
- Add models, and dbcontext
  Reference library from application project
  Using Package Manager Console, try running a Add-Migration.

Note: We have added references to Microsoft.EntityFrameworkCore + *.Tools + *.Design.Core to the application project too as suggested in other issues.
### The issue

I want to add a migration.  Why can I not do this today? Why is the tooling still in preview? Surely this should be supported by now, September 16th 2016?!?

```
PM> Add-Migration -Name InitialMigration -Context ApplicationContext -Project Procore.DB -Verbose
Using startup project 'Procore'.
Using project 'Procore.DB'
Build started...
Build succeeded.
Using application base 'C:\Users\Allan\Source\Repos\uwp\src\Procore\bin\x64\Debug\'.
No configuration file found.
Using current directory 'C:\Users\Allan\Source\Repos\uwp\src\Procore\bin\x64\Debug\'.
Microsoft.EntityFrameworkCore.Design.OperationException: Could not load assembly 'Procore.DB'. Ensure it is referenced by the startup project 'Procore'. ---> System.BadImageFormatException: Could not load file or assembly 'Procore.DB' or one of its dependencies. An attempt was made to load a program with an incorrect format.
   at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
   at Microsoft.EntityFrameworkCore.Design.AssemblyLoader.Load(String assemblyName)
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.<>c__DisplayClass5_0.<.ctor>b__2()
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.<>c__DisplayClass5_0.<.ctor>b__2()
   at Microsoft.EntityFrameworkCore.Internal.LazyRef`1.get_Value()
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.<>c__DisplayClass5_0.<.ctor>b__5()
   at Microsoft.EntityFrameworkCore.Internal.LazyRef`1.get_Value()
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.AddMigrationImpl(String name, String outputDir, String contextType)
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.AddMigration.<>c__DisplayClass0_1.<.ctor>b__0()
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.OperationBase.<>c__DisplayClass3_0`1.<Execute>b__0()
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.OperationBase.Execute(Action action)
Could not load assembly 'Procore.DB'. Ensure it is referenced by the startup project 'Procore'.
```
### Further technical details

EF Core version: 1.0.0  (also tried with 1.0.1)
Operating system: Windows 10 AU
Visual Studio version: VS 2015 Update 3 and the n number of KB updates through today.

**Comment by allanrsmith** [comment]
http://stackoverflow.com/questions/23666716/enable-migrations-in-x64-project-gets-system-badimageformatexception
https://entityframework.codeplex.com/workitem/2273

Why is this not a big red banner on the repo home page?  holy crap.

**Comment by rowanmiller** [comment]
The x64 issue is tracked by https://github.com/aspnet/EntityFramework/issues/1074 and has already been fixed for 1.1 (and will ship in 1.1-preview1).

> Why can I not do this today? Why is the tooling still in preview? Surely this should be supported by now, September 16th 2016?!?

The main issue with tooling is the moving platform that we are building on top of. With the transition from DNX to .NET CLI and then MSBuild consolidation, we've had to rewrite our tools on top of each of these. With the MSBuild consolidation still very much in progress, we are still trying to build on top of something as it is developed. To that end, the tools will stay pre-release until the components that they build on are stable.
